src/bounds.jl

```diff
@@ -146,3 +146,51 @@ a definite sign.
 
 """
 bound_taylor1(fT::TaylorModel1, I=domain(fT)::Interval) = bound_taylor1(polynomial(fT), I)
+
+"""
+    linear_dominated_bounder(fT::TaylorModel1, ϵ=1e-3::Float, max_iter=5::Int)
+
+Compute a tighter polynomial bound for the Taylor model `fT` by the linear
+dominated bounder algorithm. The linear dominated algorithm is applied until
+the bound of `fT` gets tighter than `ϵ` or the number of steps reachs `max_iter`.
+"""
+function linear_dominated_bounder(fT::TaylorModel1; ϵ=1e-3, max_iter=5)
+    d = 1.
+    Dn = fT.dom
+    Dm = Dn
+    Pm = Taylor1(copy(fT.pol.coeffs))
+    bound = interval(0.)
+    n_iter = 0
+    while d > ϵ && n_iter < max_iter
+        if n_iter == 0
+            x0 = mid(Dn) - mid(fT.x0)
+        else
+            x0 = mid(Dn) - mid(Dm)
+        end
+        c = mid(Dn)
+        Pm = Taylor1(copy(Pm.coeffs))
+        update!(Pm, x0)
+        linear = Taylor1(copy(Pm.coeffs))
+        linear[2:end] = zero(linear[0])
+        non_linear = Taylor1(copy(Pm.coeffs))
+        non_linear[0:1] = zero(non_linear[0])
+        Li = linear[1]
+        I1 = linear(Dn - c)
+        Ih = non_linear(Dn - c)
+        bound = I1.lo + Ih
+        d = diam(bound)
+        n_iter += 1
+        if Li == 0
+            break
+        elseif Li > 0
+            new_hi = minimum([Dn.lo + (d / abs(Li)), Dn.hi])
```

Comment:
I think you can avoid allocating the vector if you use `min` instead of `minimum`:
```julia
new_hi = min(Dn.lo + (d / abs(Li)), Dn.hi)
```